examples/stac-collection-pmtiles-vector.js

```diff
@@ -0,0 +1,57 @@
+import Map from 'ol/Map.js';
+import OSM from 'ol/source/OSM.js';
+import STAC from '../src/ol/layer/STAC.js';
+import TileLayer from 'ol/layer/WebGLTile.js';
+import View from 'ol/View.js';
+import proj4 from 'proj4';
+import {register} from 'ol/proj/proj4.js';
+
+register(proj4); // required to support source reprojection
+
+const layer = new STAC({
+  displayWebMapLink: 'pmtiles',
+  data: {
+    'stac_version': '1.0.0',
+    'stac_extensions': [
+      'https://stac-extensions.github.io/web-map-links/v1.1.0/schema.json',
+    ],
+    'type': 'Collection',
+    'id': 'Firenze',
+    'description': 'Vector data as PMTiles for Firenze, Italy',
+    'license': 'ODbL',
+    'attribution': 'PMTiles',
```

Comment:
change to `© OpenStreetMap Contributors`

Reply:
Thanks, done.